packages/components/src/snackbar/list.js

```diff
@@ -3,19 +3,44 @@
  */
 import classnames from 'classnames';
 import { omit, noop } from 'lodash';
-import { useTransition, animated } from 'react-spring/web.cjs';
+import { motion, AnimatePresence } from 'framer-motion';
```

Comment:
After reading the doc, it seems swapping `motion` here with `m` improves the bundle size, did you try it?

Reply:
Oh I guess, it would force us to add the `LazyMotion` provider, which is not great for packages like you said.